src/components/footer/githubMobile.jsx

```diff
@@ -0,0 +1,24 @@
+import githubIcon from './githubIcon.png';
+
+const team = [
+  {
+    link: 'https://github.com/WildCodeSchool/p2-g2-sept21-lille-react-Videotheque',
+    memberName: 'KeepN Team',
+  },
+];
+
+const GithubMobile = () => {
+  return (
+    <div className="teamProfile">
+      {team.map((e) => (
```

Comment:
This conversation is not close, is it ? You don't `.map()` a one-element-array, it makes no sense ;-)

Reply:
This conversation is closed. Yes, it makes no sense :) This component is deleted.

Reply:
Yes, this conversation is closed. It makes no sense, off course :) This component is deleted.